pkg/volume/glusterfs/glusterfs.go

```diff
@@ -376,13 +377,33 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 		}
 
 	}
-
 	options = append(options, "backup-volfile-servers="+dstrings.Join(addrlist[:], ":"))
 
+	//fetch client version.
+	mountBinaryVer := ""
+	cmdOut, err := b.exe.Command(linuxGlusterMountBinary, "-V").CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("Failed to get binary %s version", linuxGlusterMountBinary)
```

Comment:
The only possibility which I can think of an error here, the mount binary itself is not available, so the return.